packaging/rpm/xpra.spec

```diff
@@ -828,6 +835,7 @@ fi
 %post -n %{package_prefix}-client-gtk3
 /usr/bin/update-desktop-database &> /dev/null || :
 /bin/touch --no-create %{_datadir}/icons/hicolor &>/dev/null || :
+echo "Enable the '%{gnome_shell_extension}' extension for keyboard layout management"
```

Comment:
Why is this here?

Reply:
It's just a reminder. The extension should be enabled manually because all newly installed extensions are disabled by default. When the extension is disabled, Xpra will log an error for the D-Bus query and moves on (similar to the now restricted **Eval** method in the **safe** mode).

And for the same reason, I thought that following explanation in the README.md file may be useful for end-users:
> This extension may be enabled by `/usr/bin/gnome-extensions enable input-source-manager@xpra_org` command or alternatively, it may be enabled manually by searching for the `Extensions` (by pressing the `Super` key) and finding that extension in the filtered list.